site/content/docs/5.1/utilities/flex.md

```diff
@@ -10,12 +10,12 @@ toc: true
 
 Apply `display` utilities to create a flexbox container and transform **direct children elements** into flex items. Flex containers and items are able to be modified further with additional flex properties.
 
-{{< example >}}
-<div class="d-flex p-2 bd-highlight">I'm a flexbox container!</div>
+{{< example class="bd-example-flex" >}}
+<div class="d-flex p-2">I'm a flexbox container!</div>
```

Comment:
Couldn't we have the same behavior as in CSS grid ?

Reply:
Bootstrap